### DISCUSSION TITLE
Add Read Only Calculated Fields

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14768] [How to]
Hi,
I would to create:
1. Calculated Fields (like sum, multiplication...) with immediate result
2. These Fields must be read-only

How to do that?
Very Thanks

**Comment by mariuszkrzaczkowski** [comment]
new field + displayType changed to 2 + workflow

**Reply by espe808** [reply]
Thanks!

**Reply by espe808** [reply]
Sorry but I have not this option (displayType changed to 2)

**Reply by espe808** [reply]
![sdgasdfkasj](https://user-images.githubusercontent.com/85027316/120647095-81f05e80-c47a-11eb-91d3-674c69ccec13.JPG)

**Reply by espe808** [reply]
@mariuszkrzaczkowski could tell me Why I have not this option?
thanks
